src/components/ZoomHandler.component.jsx

```diff
@@ -30,10 +30,14 @@ const ZoomHandler = React.createClass({
 		}
 	},
 	onZoomEnd() {
-		this.props.setZoom(event.transform);
+		if (!isNaN(event.transform.k) && !isNaN(event.transform.x) && !isNaN(event.transform.y)) {
```

Comment:
did this because it happen that on initialisation event is returnerd with NaN value set on x/y, guess this is because canvas size is not yet determined by the browser (code initialisation is faster in some case than layout composition)